data/utils/split_data.py

```diff
@@ -14,64 +14,63 @@
 from constants import DATASETS, SEED_FILES
 
 def create_jsons_for(user_files, which_set, max_users, include_hierarchy):
-    '''used in split-by-user case'''
+    """used in split-by-user case"""
     user_count = 0
     json_index = 0
     users = []
-    if include_hierarchy:
```

Comment:
Getting rid of these conditional makes it so hierarchies is no longer initialized anywhere. This probably didn't break anything, as I believe we don't use hierarchies yet.